### ISSUE TITLE
version `glibc_2.14' not found centos

### ISSUE THREAD

**Issue by bydbest** [danielgtaylor/jpeg-archive#59]
i am getting error version `glibc_2.14' not found centos when i checked with yum it shows 2.12 .i tried to change version no in bash file but it is unreadable. can you please  tell which release uses 2.12 glibs .i am using centos 6.8 and i checked the latest release through yum update and in their repo. its 2.12 only.
i can see only bash file that too without .sh extension. plz tell i have to make frm source or jpeg-archive-2.1.1-linux.tar.bz2
plz help

**Comment by lfos** [comment]
Sorry, but it is quite hard to parse your bug report. Does the issue persist? Did you compile jpeg-archive from source or did you install it via your package manager? When exactly does the error occur?

**Comment by lfos** [comment]
Closing for now. Please provide more details...
